docs/MVS_Outputs.rst

```diff
@@ -62,6 +62,18 @@ The LCOElectricity are then calculated with:
 
 ** The content of this section was copied from the conference paper handed in to CIRED 2020**
 
+Levelized Cost of Energy of Asset (LCOE_ASSET)
+################################
+
+This KPI measures the cost of generating 1 kWh for each asset in the system. It can be used to assess and compare the available alternative methods of energy production. The levelized cost of energy of an asset (LCOE_ASSET) is usually obtained looking at the lifetime costs of building and operating the asset per unit of total energy throughput of an asset over the assumed lifetime [currency/kWh].  
+
+Since not all assets are production assets, the MVS distinguishes between the type of assets. For assets in energyConversion and energyProduction the MVS calculates the LCOE_ASSET by dividing the total annuity of the asset by the total flow.
```

Comment:
```suggestion
Since not all assets are production assets, the MVS distinguishes between the type of assets. For assets in energyConversion and energyProduction the MVS calculates the LCOE_ASSET by dividing the total annuity $a_i$ of the asset $i$ by the total flow $\sum{t} E(t)$.
```